tests/ftpserver.pl

```diff
@@ -2867,9 +2868,14 @@ sub customize {
             logmsg "FTPD: read POSTFETCH header data\n";
             $postfetch = $1;
         }
+        elsif($_ =~ /SLOWDOWNDATA/) {
+            $ctrldelay=0;
+            $datadelay=0.005;
+            logmsg "FTPD: send response data with 0.01 sec delay per byte\n";
+        }
         elsif($_ =~ /SLOWDOWN/) {
-            $ctrldelay=1;
-            $datadelay=1;
+            $ctrldelay=0.005;
+            $datadelay=0.005;
             logmsg "FTPD: send response with 0.01 sec delay between each byte\n";
```

Comment:
This message doesn't match the actual delay.

Reply:
will adjust